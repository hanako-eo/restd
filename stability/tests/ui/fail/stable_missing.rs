#![feature(staged_api)]
#![stable(feature = "root", since = "1.0.0")]
#![allow(internal_features)]

#[stability::stable(feature = "test")]
fn test() {}

fn main() {}
