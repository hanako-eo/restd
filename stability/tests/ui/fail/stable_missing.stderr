error: since need to be set
 --> tests/ui/fail/stable_missing.rs:5:1
  |
5 | #[stability::stable(feature = "test")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `stability::stable` (in Nightly builds, run with -Z macro-backtrace for more info)
